x-pack/plugins/enterprise_search/public/applications/shared/role_mapping/user_flyout.tsx

```diff
@@ -75,7 +77,12 @@ export const UserFlyout: React.FC<Props> = ({
         <EuiButtonEmpty onClick={closeUserFlyout}>{CANCEL_BUTTON_LABEL}</EuiButtonEmpty>
       </EuiFlexItem>
       <EuiFlexItem grow={false}>
-        <EuiButton disabled={disabled} onClick={handleSaveUser} fill>
+        <EuiButton
+          disabled={disabled || formLoading}
+          isLoading={formLoading}
```

Comment:
the `isLoading` prop alone automatically handles disabling the button, no need to add an extra conditional!

```suggestion
          disabled={disabled}
          isLoading={formLoading}
```

Reply:
Fixed in both places. Thanks!